CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is (loosely) based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
-## [Unreleased]
+## [v1.3.0] - 2022-09-20
```

Comment:
```suggestion
## [1.3.0] - 2022-09-20
```

Either the `v` needs to removed from the headings, or it needs to be added to the links at the bottom.